frontend/src/components/FormReview/index.tsx

```diff
@@ -19,56 +58,31 @@ const FormReview = () => (
       <Subtitle text="Review data filled in the form (view-only)" />
     </div>
     <div>
-      <Accordion className="steps-accordion">
-        <AccordionItem
-          title={(
-            <TitleAccordion
-              title="Collection"
-              description="Review collection information"
-            />
-          )}
-        />
-        <AccordionItem
-          title={(
-            <TitleAccordion
-              title="Ownership"
-              description="Review ownership information"
-            />
-          )}
-        />
-        <AccordionItem
-          title={(
-            <TitleAccordion
-              title="Interim storage"
-              description="Review interim storage information"
-            />
-          )}
-        />
-        <AccordionItem
-          title={(
-            <TitleAccordion
-              title="Orchard"
-              description="Review orchard information"
-            />
-          )}
-        />
-        <AccordionItem
-          title={(
-            <TitleAccordion
-              title="Parent tree and SMP"
-              description="Review parent tree and SPM information"
-            />
-          )}
-        />
-        <AccordionItem
-          title={(
-            <TitleAccordion
-              title="Extraction and storage"
-              description="Review extraction and storage information"
-            />
-          )}
-        />
-      </Accordion>
+      {
+        mockFormData.length
+          ? (
+            <Accordion className="steps-accordion">
+              {
+              mockFormData.map((data) => (
+                <AccordionItem
+                  key={data.id}
+                  title={(
+                    <TitleAccordion
+                      title={data.title}
+                      description={data.description}
+                    />
+                  )}
+                />
+              ))
+            }
```

Comment:
Indentation is a bit odd here, the opening bracket on line 65 is not aligned with the closing one on line 77, an additional tab from line 66 to line 77 should fix it.